### ISSUE TITLE
mobile/ text alternatives for non-text content

### ISSUE THREAD

**Issue by serdarbahar** [bounswe/bounswe2025group5#563]
### Task Description
All non-text content in the applications should have a text alternative according to the accessibility requirements. This issue will deal with implementing the feature in mobile and confirming that all non-text content have proper text alternatives. The relevant [requirement](https://github.com/bounswe/bounswe2025group5/wiki/Requirements-of-CMPE451) is 2.1.2.5.

### Why Is This Needed?
- To follow accessibility requirements throughout our project

### Acceptance Criteria
- [ ] **all** non-text content have text alternatives
- [ ] text alternatives are clear and unambiguous

### Deadline
20.11.2025 - 23:59

**Comment by serdarbahar** [comment]
I've created a [PR](https://github.com/bounswe/bounswe2025group5/pull/566) regarding my changes.

**Comment by kerimkasar** [comment]
It looks like the PR is merged. I am closing the issue.
